requirements.txt

```diff
@@ -19,8 +19,9 @@ webargs==0.18.0
 slacker==0.8.6
 raven[flask]==5.8.1
 ujson==1.33
-pyelasticsearch==1.4
```

Comment:
Yikes not sure how I missed that. I'll convert it to elasticsearch-py

In taking a look, I'm seeing some duplicate code. It looks like the legal commands were never removed from `manage.py` https://github.com/18F/openFEC/blob/develop/manage.py#L315